portality/static/widget/fixed_query_dev.js

```diff
@@ -1,12 +1,56 @@
+"use strict";
 (function() {
-    // Load the widget's js scripts then HTML content. We assume jQuery is available.
+// Localize jQuery variable
+let jQuery;
+
+/******** Load jQuery if not present *********/
+if (window.jQuery === undefined || window.jQuery.fn.jquery !== "1.9.1") {
+    let script_tag = document.createElement("script");
+    script_tag.setAttribute("type","text/javascript");
+    script_tag.setAttribute("src",
+        "http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js");
+    if (script_tag.readyState) {
+      script_tag.onreadystatechange = function () { // For old versions of IE
+          if (this.readyState === "complete" || this.readyState === "loaded") {
+              scriptLoadHandler();
+          }
+      };
+    } else { // Other browsers
+      script_tag.onload = scriptLoadHandler;
+    }
+    // Try to find the head, otherwise default to the documentElement
+    (document.getElementsByTagName("head")[0] || document.documentElement).appendChild(script_tag);
+} else {
+    // The jQuery version on the window is the one we want to use
+    jQuery = window.jQuery;
+    main();
+}
+
+/******** Called once jQuery has loaded ******/
+function scriptLoadHandler() {
+    // Restore $ and window.jQuery to their previous values and store the
+    // new jQuery in our local jQuery variable
+    jQuery = window.jQuery.noConflict(true);
+    // Call our main function
+    main();
+}
+
+/******** Our main function ********/
+function main() {
     jQuery(document).ready(function($) {
-        $('#doaj-fixed-query-widget').load(doaj_url + "/static/widget/fixed_query_body_dev.html");
+        let url = doaj_url + "/static/widget/fixed_query_body_dev_edges.html";
```

Comment:
It would be lovely to find a DRYer way of doing this, so we don't have to have different versions of the files for different environments.  I don't know if that's possible - we obviously didn't find a way of doing it before, but perhaps we can pull all this stuff out at least to an environment-localisation file like `widget-dev.js` which contains all the dev-specific urls